### DISCUSSION TITLE
Does one need elevated privileges to access Copilot API metrics?

### DISCUSSION THREAD

**Discussion by kiranparuchuru** [community/community#136391] [Copilot Conversations] [answered]
**Select Topic Area**

Question

**Body**

Hi.  

I am normal GitHub user, and do not have elevated privileges/admin right etc. 

The reference page indicates:
_"Organization owners, and owners and billing managers of the parent enterprise, can view Copilot usage metrics."_
Reference: https://docs.github.com/en/rest/copilot/copilot-usage?apiVersion=2022-11-28

Clarification.  Can an admin create the OAuth token for me with necessary scopes, 
_"OAuth app tokens and personal access tokens (classic) need either the manage_billing:copilot, read:org, or read:enterprise scopes to use this endpoint."_

so I can utilize the APIs?  Will this work, or do I really need be an organization owner/billing manager and then create the OAuth myself?

Thanks in advance.

**Comment by GabrielAlmeidag** [comment, marked as answer]
@kiranparuchuru 😺👋

Even with an OAuth token with the necessary scopes (manage_billing:copilot, read:org, or read:enterprise), you will still need the appropriate permissions to view Copilot usage metrics. According to GitHub's documentation, only organization owners and billing managers of the controlling company can access these metrics.

Therefore, if you are not an owner or billing manager of the organization, even if an administrator creates the OAuth token for you, you still won’t be able to access this information. To gain access, you will need someone with the appropriate permissions to provide the metrics or to receive the status of an owner or billing manager in the organization.
